examples/application_profile/config/apps/app-profile-herbs.yaml

```diff
@@ -0,0 +1,36 @@
+app-profile:
+  - name: app-tf-minio
+    version: 1.0.0
+    context: project
+    description: created from tf-modules
+    cloud: all
+    pack:
+      - name: minio-operator-stage
+        type: ${type_operator_instance}
+        registry_uid: ${db_registry_uid}
```

Comment:
input should be a name rather than uid

Reply:
Addressed , Now by giving name for registry and for pack name, version is fine. modules will pull required uid